ui/components/vpl/vpl-nav/_index.scss

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
+// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
+
+/**
+* @name vpl
+*
+*/
+.nds-subnav__wrapper{
+    position:relative;
+   .nds-subnav__fullWidth{
+   	width:100%;
+   }
+ .nds-subnav__active{
+       display:flex;
+    -moz-flex-direction: column;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    height: 100%;
+    border-right: 0.0625rem solid rgba($color-black-primary, 0.1);
+    border-left: 0;
+    padding: 0.625rem;
+    cursor:pointer;
+    border-bottom: 0.0625rem solid $card_color_background;
+    background: $card_color_background;
+        color: $color-blue-text-secondary;
+   
+      &.nds-subnav__tabSelected {
+        border-left:0.0625rem solid rgba($color-black-primary, 0.1);
+        background:$color-brand;
+        border-bottom:0;
+        color:$color-border-tab-active;
+        img{
+		-webkit-filter: brightness(0.25);
+		filter: brightness(5);
+        }
+        
+        &:hover{
+           border-bottom-color:transparent;
+       }
+    }
+    &.nds-subnav__tabSelected{
+    	.nds-subnav__cardtop{
+        vertical-align : middle;
+        margin : auto;
+       
+        }
+    }
+     .nds-subnav__image{
+	        display: flex;
+	        justify-content: center;
+	        }
+ }
+ 
+ .nds-subnav__active{
+ 	
+		.nds-subnav__cardValues h2
+		{
+		font-size: 0.625rem
+
+		}
+        .nds-subnav__cardValues h2:first-child
+        {
+            font-size: 1.125rem;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            display: -webkit-box;
```

Comment:
We should change this to flex and use the proper flexbox rules. From https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex:

> This property is a non-standard extension. There was an old draft of the CSS3 Flexbox specification that defined a box-flex property, but that draft has since been superseded.